kiva/tests/test_gl_drawing.py

```diff
@@ -33,11 +33,6 @@ def test_star_clip(self):
         # FIXME: overriding test since it segfaults
         DrawingImageTester.test_star_clip(self)
 
-    @unittest.expectedFailure
-    def test_text_clip(self):
-        # gl graphics context does not clip text properly (#165).
-        DrawingImageTester.test_text_clip(self)
-
```

Comment:
I think you need to skip this test instead of remove it from this derived class.